Modules/PSFramework.md

```diff
@@ -0,0 +1,16 @@
+# Name: PSframework
+
+## Description:
+Module designed to provide powerful infrastructure features to powershell scripters. Whether it is to provide features, save time, improve user experience or drive manageability of PowerShell itself.
+
+| Works with | Author(s) | Project URL(s) | Creation Date | Tags |
+|------------|--------|-------------------|---------------|------|
+| PowerShell | Friedrich Weinmann (<a href="http://allthingspowershell.blogspot.de" target="_blank">b</a> \| <a href="https://twitter.com/FredWeinmann" target="_blank">t</a>) | [Github](https://github.com/PowershellFrameworkCollective/psframework) \| [official project site](https://github.com/PowershellFrameworkCollective/psframework) | 4 / 2017 | Tools, Infrastructure, Framework, PowerShell, Logging, Management |
```

Comment:
Please leave just GitHub entry for the Project URL(s) once they are the same.
Set the Creation Date as "April 2017"